projects/fsastorefrontlib/src/cms-components/seller-dashboard/seller-dashboard.component.ts

```diff
@@ -16,17 +22,37 @@ export class SellerDashboardComponent {
   constructor(
     protected consentConnector: ConsentConnector,
     protected userIdService: UserIdService,
-    protected userAccountFacade: UserAccountFacade
+    protected userAccountFacade: UserAccountFacade,
+    protected renderer: Renderer2
   ) {}
 
+  @ViewChild('dashboardOverviewContent') dashboardOverviewContent: ElementRef;
+  @ViewChild('createCustomer') createCustomer: ElementRef;
   seller$: Observable<FSUser> = this.userAccountFacade.get();
   customers$: Observable<OBOCustomerList> = this.userIdService.getUserId().pipe(
     take(1),
     switchMap(userId => this.consentConnector.getOBOCustomerList(userId))
   );
   dashboardListVisible = false;
+  showCustomerForm = false;
 
   showDashboardList() {
     this.dashboardListVisible = true;
   }
+  showAddCustomerForm() {
+    this.showCustomerForm = true;
+    this.renderer.removeClass(
+      this.dashboardOverviewContent.nativeElement,
+      'slide-out'
+    );
+    this.renderer.addClass(this.createCustomer.nativeElement, 'slide-in');
+  }
+  addedUser(action: string) {
+    this.showCustomerForm = false;
+    this.renderer.addClass(
+      this.dashboardOverviewContent.nativeElement,
+      'slide-out'
+    );
+    this.renderer.removeClass(this.createCustomer.nativeElement, 'slide-in');
+  }
```

Comment:
Maybe all of this show/hide logic could have been used in the template? Can it be done without `renderer` just by checking true/false for a `showCustomerForm`, or something like that?